includes/results.inc

```diff
@@ -1021,7 +1021,11 @@ class IslandoraSolrFacets {
       $link['count'] = $count;
       $link['link_plus'] = '<a' . drupal_attributes($attributes['plus']['attr']) . '>+</a>';
       $link['link_minus'] = '<a' . drupal_attributes($attributes['minus']['attr']) . '>-</a>';
-      $buckets[] = $link;
+      $buckets[$bucket] = $link;
+    }
+
+    if (isset($this->settings['solr_field_settings']['sort_by']) && $this->settings['solr_field_settings']['sort_by'] == 'index') {
+      ksort($buckets);
```

Comment:
@adam-vessey Oops, that is my mistake. I did try that until I ran in issue of sorting by one value then displaying a different one (ie. replace with object label). I'll update the ticket with the issue I had with that method. 

I'll also work on some more descriptive wording for the options (maybe switch to radio buttons). 

As for transformations, I see what you are suggesting. Which isn't a bad idea, but would require a wholesale change to the way facets are created...I think. If not, I'm open to it. But I might need a little direction.